PhotoFeed/app/src/main/java/com/mobileproto/jovanduy/photofeed/DbAccessor.java

```diff
@@ -0,0 +1,98 @@
+package com.mobileproto.jovanduy.photofeed;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.util.Log;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+/**
+ * Class to modify the database
+ */
+public class DbAccessor {
```

Comment:
This is really a personal design choice, but can you explain to me why you chose to separate this class from the class the FeedReaderDbHelper and not just put these methods in that class? @jovanduy